### PR TITLE
Test udpate

### PR DESCRIPTION
**Pull request by masondoom**
???REALLY

### CODE REVIEW

File: src/main/kotlin/com/masondoom/rpntool/Main.kt [hunk 1]
```diff
@@ -5,7 +5,8 @@ import com.masondoom.rpntool.core.calculator.Calculator
 import com.masondoom.rpntool.core.utils.InputParser
 
 fun main() {
```
Comment (hibikisea) [inline src/main/kotlin/com/masondoom/rpntool/Main.kt:7]: Pls add comments

File: src/main/kotlin/com/masondoom/rpntool/Main.kt [hunk 2]
```diff
@@ -5,7 +5,8 @@ import com.masondoom.rpntool.core.calculator.Calculator
...
+
     val calculator = Calculator()
```
Comment (hibikisea) [inline src/main/kotlin/com/masondoom/rpntool/Main.kt:9]: pls use strong type instead of type inference

**Review by hibikisea — CHANGES_REQUESTED** [review]
Gge

File: src/main/kotlin/com/masondoom/rpntool/Main.kt [hunk 3]
```diff
@@ -5,7 +5,8 @@ import com.masondoom.rpntool.core.calculator.Calculator
 import com.masondoom.rpntool.core.utils.InputParser
 
 fun main() {
+
```
Comment (masondoom) [inline src/main/kotlin/com/masondoom/rpntool/Main.kt:8]: Adding a new line

**Pull request closed without merging**